### ISSUE TITLE
Timer too fast.

### ISSUE THREAD

**Issue by ad77076** [shubhambathe1/react-native-countdown-timer-hooks#7]
Really love the package, it's very helpful but the timer is ticking too fast. Very fast. The seconds are running like miliseconds. If you have a solution to this problem, please let me know asap. Thank You.

**Comment by shubhambathe1** [comment]
hello @ad77076 thanks for reaching out. But can you elaborate your issue a bit more, because the timer does not tick very fast as you have mentioned. It ticks only after 1 sec for each and every tick. It will be great if you can compare the timer component time with a real stopwatch. Please try this and let me know if you still have the issue.

**Comment by ad77076** [comment]
Yes, I've tried with a real timer and 30secs of real time gets completed in 10-12 real secs of your timer which has been set for 30seconds only. And the problem is it's not fixed, sometimes 30secs gets completed in 10 secs and sometimes in 15secs. Anyways thanks for replying so fast mate, I've used a different package. Happy Coding, Hail Hydra !

**Comment by shubhambathe1** [comment]
@ad77076 Just check if you have passed any "delay" props to the timer component. If you have passed it make sure it's set to 1000ms so the timer will tick at 1 sec delay only. You might have set a delay value less than 1000ms that's why it's ticking faster. I hope it helps.

**Comment by ad77076** [comment]
Timer still inaccurate !

 <CountDownTimer
                ref={refTimer}
                delay={1000}
                timestamp={30}
                timerCallback={()=>{
                  
                }}
                containerStyle={styles.timerContainerStyle}
                textStyle={{...styles.timerText,color: colorScheme === 'dark' ? '#fafafa' : '#000000'}}/>

This is the code I'm using and it still completes 7secs earlier than it should.

**Comment by shubhambathe1** [comment]
Here I am sharing my screen recording. The 30 sec timer working fine for me. 

[Screencast from 09-04-24 04:48:29 PM IST.webm](https://github.com/shubhambathe1/react-native-countdown-timer-hooks/assets/31159892/82a72344-747d-4751-88d7-c5449ab2a670)

If possible can you share your screen recordings, so I will be able to debug the issue.
